apps/expo/src/styles.css

```diff
@@ -1,6 +1,51 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@import "tailwindcss";
+
+@theme {
+  --color-border: hsl(var(--border));
```

Comment:
These variables just work using `react-native-css`?

I'm asking because I don't really see how they are added to the Expo app in this case now that the Tailwind config file was removed.